js/constants.js

```diff
@@ -0,0 +1,19 @@
+//create button
+export const createTask = document.getElementById("createButton");
+
+//content  task list
```

Comment:
I guess this comments are safe to remove as we can easily understand the below two line by there name, right?